### DISCUSSION TITLE
Your login session has expired

### DISCUSSION THREAD

**Discussion by Droppisalt** [dani-garcia/vaultwarden#2011] [Q&A] [answered]
Hey, I just installed Vaultwarden using the docker installation found on the main Github. However, after creating an account, I can't seem to stay logged in for more than a millisecond. I see my dashboard and instantly get logged out and get the error below. Any idea what might be causing this?

logs show this (email censored)
[2021-09-30 01:44:11.544][response][INFO] POST /identity/connect/token (login) => 200 OK,
[2021-09-30 01:44:11.544][vaultwarden::api::identity][INFO] User ****@gmail.com logged in successfully. IP: 172.68.142.191

![image](https://user-images.githubusercontent.com/67336980/135371563-066f93d1-e5ca-4e96-870a-1db750daa77f.png)

**Comment by Droppisalt** [comment, marked as answer]
Leaving this here if anyone gets the same error. Accidentally used access lists in Nginx and this caused the problem. Not sure if this is intended behaviour but it now works

**Reply by SamTV12345** [reply]
Did you solve the issue with enabled access lists? I don't feel comfortable having my instance publicly available.

**Reply by klaas0** [reply]
If you use Nginx Proxy Manager, you can select the option "Pass Auth to Host" for it to work.
